.github/workflows/get-published-image-names.yaml

```
@@ -0,0 +1,41 @@
+name: Get oci-images names from Dockerhub
+
+on:
+  workflow_call:
+    outputs:
+      images-names:
+        description: A JSON array of images that Paths in this repository where rockcraft projects are stored
```

Should we reword this a bit? I think I understand what this says but it's a bit confusing e.g.
```suggestion
        description: A JSON array of all images that have their rockcraft project stored in this repository
```